app/src/format.py

```diff
@@ -102,19 +102,47 @@ def _format_to_accordion_html(document: Document, score: float) -> str:
 def _format_to_accordion_group_html(documents: OrderedDict[Document, list[ChunkWithScore]]) -> str:
     global _accordion_id
     html = ""
+    citation_number = 1
     for document in documents:
-        internal_citation = ""
+        citations = ""
         _accordion_id += 1
-        for index, chunk in enumerate(documents[document], start=1):
-            chunk_lines = chunk.chunk.content.splitlines()
-            formatted_chunk = " ".join(chunk_lines)
-            formatted_chunk = re.sub(r"\t+", "", formatted_chunk).strip()
-            formatted_chunk = (
-                f"<p>{_replace_bem_with_link(formatted_chunk)} </p>" if formatted_chunk else ""
+
+        citation_number_start = citation_number
+
+        for chunk_with_score in documents[document]:
+            chunk = chunk_with_score.chunk
+            formatted_chunk = _replace_bem_with_link(chunk.content)
+
+            # Adjust markdown for lists so Chainlit renders correctly
+            formatted_chunk = formatted_chunk.replace("    - ", "- ")
```

Comment:
Should this be replacing only the first-level of list items so that nested list items aren't unnested?
```suggestion
            formatted_chunk = formatted_chunk.replace(r"^    - ", "- ")
```

Reply:
Good call. Ultimately we should fix this in the ingestion code and remove the code here (hopefully before or at the same time as adding support for nested links), but better safe than sorry